### DISCUSSION TITLE
How to trigger GitHub Actions on retag

### DISCUSSION THREAD

**Discussion by CAMOBAP** [community/community#26202] [Actions]
<p>Recently I faced a problem I put a tag on a wrong commit &amp; release flow failed.</p>
<p>I decided to retag with <code>git tag -f v1.2.3 &amp; git push -f --tags</code> but nothing happens</p>
<p>After this I decided to delete tags both locally &amp; remotely and push it again:</p>
<p> - <code>git tag -d v1.2.3</code></p>
<p> - <code>git push --delete origin v1.2.3</code></p>
<p> - <code>git tag v1.2.3</code></p>
<p>But again nothing happens</p>
<p>How can I trigger a workflow for the same tag again?</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Checked on my side, after i delete the tag and retag, the workflow can be triggered successfully. After you delete the tag locally and remotely, please check on github webpage(<a href="https://github.com/%7Borg%7D/%7Brepo%7D/tags" rel="nofollow noopener">https://github.com/{org}/{repo}/tags</a>) to confirm it’s deleted. And after you recreate the tag locally, please also push it to remote to trigger the workflow. git tag v1.2.3 git push origin --tags</p>

**Comment by CAMOBAP** [comment]
<p>It was my mistake I did <code>git tag -f ...</code> instead <code>git tag -d ...</code></p>
<p>Thanks for help</p>
